spec/integ/rendezvous/MSC4108SignInWithQR.spec.ts

```diff
@@ -158,18 +156,18 @@ describe("MSC4108SignInWithQR", () => {
 
             const ourChannel = new MSC4108SecureChannel(ourMockSession);
             const qrCodeData = QrCodeData.from_bytes(
-                await ourChannel.generateCode(QrCodeMode.Reciprocate, client.getHomeserverUrl()),
+                await ourChannel.generateCode(QrCodeMode.Reciprocate, client.getDomain()!),
             );
             const opponentChannel = new MSC4108SecureChannel(opponentMockSession, qrCodeData.public_key);
 
             ourLogin = new MSC4108SignInWithQR(ourChannel, true, client);
             opponentLogin = new MSC4108SignInWithQR(opponentChannel, false);
         });
 
-        it("should be able to connect with opponent and share homeserver url & check code", async () => {
+        it("should be able to connect with opponent and share server name & check code", async () => {
             await Promise.all([
                 expect(ourLogin.negotiateProtocols()).resolves.toEqual({}),
-                expect(opponentLogin.negotiateProtocols()).resolves.toEqual({ homeserverBaseUrl: client.baseUrl }),
+                expect(opponentLogin.negotiateProtocols()).resolves.toEqual({ homeserverBaseUrl: client.getDomain() }),
```

Comment:
`homeserverBaseUrl` should probably be renamed too?

Reply:
Indeed, awaiting the rust bindings before I make any changes here